src/adjoint_common.jl

```diff
@@ -327,11 +327,10 @@ end
 function generate_callbacks(sensefun, g, λ, t, callback, init_cb)
 
   reverse_cbs = setup_reverse_callbacks(callback,sensefun.sensealg)
-  sensefun.discrete || return reverse_cbs
+  sensefun.discrete || return reverse_cbs, nothing
```

Comment:
oh...

Reply:
This was added because of the error from the earlier CI run https://github.com/SciML/DiffEqSensitivity.jl/runs/1696580565

Reply:
I'm saying, yeah oops, I should've noticed this one immediately haha.